fcsa_service_request-1.0.0-resolved.json

```diff
@@ -0,0 +1,668 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "FCSA Service Request",
+    "description": "FCSA Service Request",
+    "contact": {
+      "email": "dee@opencitieslab.org"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    },
+    "version": "2.1"
+  },
+  "servers": [
+    {
+      "url": "https://qaeservices1.capetown.gov.za/coct/api/zcur-guest/login",
+      "description": "FCSA Service Request API"
+    }
+  ],
+  "tags": [
+    {
+      "name": "sap",
+      "description": "Read in data to be uploaded and prepare for post operation"
+    },
+    {
+      "name": "status",
+      "description": "Get status of stored reference numbers"
+    },
+    {
+      "name": "upload_with_image",
+      "description": "Get status of stored reference numbers"
+    }
+  ],
+  "paths": {
+    "/get_sap_cookie": {
+      "get": {
+        "tags": [
+          "sap"
+        ],
+        "summary": "Get MYSAPSSO2 cookie.",
+        "description": "Pass in the header infomation containing the X-Service public key\n",
```

Comment:
It's not clear to me from this description what this operation does - I suspect it's rather something like *based upon the header information passed in, returns an authentication cookie*